src/reset.css

```diff
@@ -0,0 +1,69 @@
+/* #region Normalize */
```

Comment:
I think, there is no reason to make changes to unnecessary elements

Reply:
It make sense to default `reset.css` file